testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -385,6 +397,20 @@ impl SignCoordinator {
                 debug!("Ignoring StackerDB event for non-signer contract"; "contract" => %event.contract_id);
                 continue;
             }
+            let modified_slots = &event.modified_slots;
+
+            // Update `next_signers_bitvec` with the slots that were modified in the event
+            modified_slots.iter().for_each(|chunk| {
+                if let Ok(slot_id) = chunk.slot_id.try_into() {
```

Comment:
What's the alternative if this conversion fails?

Reply:
Hm, I'd love your input here. If we're at this point in the code, and we already know this is from a signer stackerDB contract, is `slot_id` ever expected to be > u16? My understanding is that slot_id should never be larger than the max reward set size, so I guess my thought is that this is a "if we get here, we have other problems" thing. So, maybe a panic or warn?

Reply:
Probably an `error!`, but yeah it really ought to be unreachable